packages/openapi3-parser/lib/parser/oas/parseOperationObject.js

```diff
@@ -47,6 +47,18 @@ function createTransactions(namespace, member, operation) {
       const clonedRequest = request.clone();
       clonedRequest.method = method;
 
+      const { contentType } = response;
+      if (contentType) {
+        let { headers } = clonedRequest;
+
+        if (!headers) {
+          headers = new namespace.elements.HttpHeaders();
+          clonedRequest.headers = headers;
+        }
+
+        headers.unshift(new namespace.elements.Member('Accept', contentType.toValue()));
```

Comment:
What if the response has an `Accept` header defined in the ADD?

Reply:
`Accept` header is not permitted in Parameter Object (which is where else you could try set it) https://spec.openapis.org/oas/v3.0.2#fixed-fields-9

> If in is "header" and the name field is "Accept", "Content-Type" or "Authorization", the parameter definition SHALL be ignored.

We have test to ensure it's dropped and a warning is presented:

https://github.com/apiaryio/api-elements.js/blob/master/packages/openapi3-parser/test/unit/parser/oas/parseParameterObject-test.js#L136-L146